src/main/java/me/snowdrop/licenses/sanitiser/RedHatLicense.java

```diff
@@ -32,13 +32,16 @@
 
     private String url;
 
+    private String textUrl;
+
     private Set<String> aliases;
 
     private Set<String> urlAliases;
 
     public RedHatLicense(JsonObject jsonObject) {
         this.name = jsonObject.getString("name");
         this.url = jsonObject.getString("url");
+        this.textUrl = jsonObject.getString("textUrl", null);
```

Comment:
alternatively we could do `this.textUrl = jsonObject.getString("textUrl", url);` and then always set text url in the `AliasLicenseSanitiser`.
@gytis what do you think about it?

Reply:
Maybe always having a value in textUrl would make it more clear.